source/benchmarking/benchmarking.md

```diff
@@ -410,36 +416,144 @@ Dataset size: For score purposes, the dataset size for a task is the size of the
+| Before task | Drop the `corpus` collection. Create an empty `corpus` collection with the `create` command. Construct a Collection object for the `corpus` collection to use for insertion.       |
+| Do task     | Do an ordered `insertMany` with 10 copies of the document.  DO NOT manually add an `_id` field; leave it to the driver or database. |
+| After task  | n/a                                                                                                                                                                                |
+| Teardown    | Drop the `perftest` database.                                                                                                                                                      |
+
+#### Small doc Collection BulkWrite insert
+
+Summary: This benchmark tests driver performance inserting multiple small documents to the database using the `Collection::bulkWrite` operation.
+
+Dataset: The dataset, designated SMALL_DOC consists of a JSON document with an encoded length of approximately 250
+bytes.
+
+Dataset size: For score purposes, the dataset size for a task is the size of the single-document source file (275 bytes)
+times 10,000 operations, which equals 2,750,000 bytes or 2.75 MB.
+
+| Phase       | Description                                                                                                                                                                                                                                                                                        |
+|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Setup       | Construct a MongoClient object. Drop the `perftest` database. Load the SMALL_DOC dataset into memory as a language-appropriate document type (or JSON string for C). Create a list of insert models for 10 copies of the document, allowing the driver or database to auto-assign the `_id` field. |
+| Before task | Drop the `corpus` collection. Create an empty `corpus` collection with the `create` command. Construct a Collection object for the `corpus` collection to use for insertion.                                                                                                                       |
+| Do task     | Do an ordered `Collection::bulkWrite` in the same namespace ("perftest.corpus") with the list of insert models.                                                                                                                                                                                    |
```

Comment:
Is namespace relevant here?

Reply:
Yep namespace isn't relevant here for `Collection:bulkWrite` since you can't have operations on different collections.